### PR TITLE
#28 - added configuration

### PR DESCRIPTION
**Pull request by olenagerasimova**
Part of #28 
Added class for repository config.

### CODE REVIEW

File: src/main/java/com/artipie/debian/Config.java [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+         * @param yaml Setting in yaml format
+         */
+        public FromYaml(final String name, final Optional<YamlMapping> yaml) {
+            this.name = name;
+            this.yaml = yaml.orElseThrow(
+                () -> new IllegalArgumentException(
+                    "Illegal config: `setting` section is required for debian repos"
+                )
+            );
+        }
```
Comment (genryxy) [inline src/main/java/com/artipie/debian/Config.java:83]: @olenagerasimova why is it necessary to use optional here if this field is obligatory?

**Review by genryxy — CHANGES_REQUESTED** [review]
@olenagerasimova thanks, one comment

File: src/main/java/com/artipie/debian/Config.java (hunk 1 above)
Comment (olenagerasimova) [reply src/main/java/com/artipie/debian/Config.java:83]: @genryxy `com.artipie.RepoConfig#settings` from artipie/artipie returns optional, I think that specific adapter settings (and their absens) should be handled of the adapter side, not artipie/artipie

**Review by genryxy — APPROVED** [review]
@olenagerasimova thanks, got it
@g4s8 please check/merge

**Pull request merged**